cmd/crictl/image.go

```diff
@@ -35,6 +36,20 @@ const (
 	truncatedImageIDLen = 13
 )
 
+type imageBySort []*pb.Image
```

Comment:
We don't have created time for image. :( I think it's fine to sort by tag/digest for now.

Reply:
Why compare digest first? I think we care more about tag, right?

Reply:
done.